package.json

```diff
@@ -38,31 +38,35 @@
     "babel-loader": "8.0.4",
     "chalk": "^2.4.2",
     "codecov": "^3.1.0",
-    "concurrently": "^4.1.0",
     "enzyme": "^3.8.0",
     "enzyme-adapter-react-16": "^1.7.1",
     "http-proxy-response-rewrite": "^0.0.1",
     "jest-enzyme": "^7.0.1",
     "jest-fetch-mock": "^2.1.0",
     "prettier": "1.16.2",
     "pretty-quick": "^1.8.1",
+    "stmux": "^1.6.1",
     "supertest": "^3.4.2",
     "type-coverage": "^1.8.0"
   },
   "scripts": {
     "build": "react-scripts build",
-    "dev": "cat .env.common-local .env.dev > .env.local && yarn start-dev",
+    "build-dev-config": "cat .env.common-local .env.dev > .env.local",
+    "dev": "yarn build-dev-config && yarn stmux [ 'yarn serve-dev' : 'yarn react-scripts-start' ]",
+    "dev-all": "yarn build-dev-config && yarn stmux [ 'yarn test' .. 'tsc -w' .. [ 'yarn react-scripts-start' : 'yarn serve-dev' ] ]",
```

Comment:
Can we stack all the panels instead?

```diff
diff --git a/package.json b/package.json
index 1692e77..de6b7db 100644
--- a/package.json
+++ b/package.json
@@ -53,7 +53,7 @@
     "build": "react-scripts build",
     "build-dev-config": "cat .env.common-local .env.dev > .env.local",
     "dev": "yarn build-dev-config && yarn stmux [ 'yarn serve-dev' : 'yarn react-scripts-start' ]",
-    "dev-all": "yarn build-dev-config && yarn stmux [ 'yarn test' .. 'tsc -w' .. [ 'yarn react-scripts-start' : 'yarn serve-dev' ] ]",
+    "dev-all": "yarn build-dev-config && yarn stmux [ 'yarn test' : 'tsc -w' : 'yarn react-scripts-start' : 'yarn serve-dev' ]",
     "eject": "react-scripts eject",
     "prettier": "prettier --write '**'",
     "prettier-ci": "prettier --list-different '**' || (echo '\n\nThis failure means you did not run `yarn prettier-dev` before committing\n\n' && exit 1)",

```

![screen shot 2019-01-31 at 10 49 41](https://user-images.githubusercontent.com/217628/52046017-0565b200-2546-11e9-859a-74515fe61cf9.png)